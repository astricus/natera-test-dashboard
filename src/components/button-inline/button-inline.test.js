import React from 'react';
import { shallow } from 'enzyme';
import ButtonInline from './button-inline.component';

test('should render ButtonInline component', () => {
  expect(shallow(<ButtonInline>Click me!</ButtonInline>)).toMatchSnapshot();
});
