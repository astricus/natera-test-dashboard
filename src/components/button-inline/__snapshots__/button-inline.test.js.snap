// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ButtonInline component 1`] = `
<button
  className="btn-inline "
>
  Click me!
</button>
`;
